internal/package_ota.sh

```diff
@@ -70,6 +70,31 @@ EOF
     "url": "$AGNOS_STAGING_UPDATE_URL/$FILENAME"
 EOF
 
+  if [ "$SPARSE" == "true" ]; then
```

Comment:
I thought we talked about making this a separate script that takes an agnos manifest?

Reply:
I can still write that script (to be able to process the existing manifest?), but I wanted to check whether we need to have the output in a separate manifest or not - I didn't know what that helped with

Reply:
it has to be a different manifest since openpilot's agnos updater flashes all the entries in the manifest

Reply:
do we want openpilot to use this image eventually?

Reply:
no